CRNotifications/Classes/CRNotification.swift

```diff
@@ -34,38 +34,42 @@ public class CRNotificationView: UIView, CRNotification {
         label.translatesAutoresizingMaskIntoConstraints = false
         label.font = UIFont.systemFont(ofSize: 13, weight: .semibold)
         label.textColor = .white
-        label.numberOfLines = 2
+        label.numberOfLines = 0
```

Comment:
The purpose of notifications is to inform of lightweight information. If your text cannot fit within 2 lines you are doing something wrong and should use another approach to communicate your information to your user, like an alert.